### ISSUE TITLE
Projects page in Sonar should show(restore) projects after backup

### ISSUE THREAD

**Issue by ghost** [zebrunner/sonarqube#18]
Steps:
1.Start sonarqube `./zebrunner.sh start`
 2.Open in your browser `<your_hostname>:9000/sonarqube`
 3.Login with demo credentials (admin/admin)
 4.Create some sample project in Jenkins and run test Runs
5. Run `./zebrunner.sh backup`
  6.delete sample project in Sonar
7. Restore `./zebrunner.sh restore`
 8.Restart container `./zebrunner.sh start`
9.Open in your browser `<your_hostname>:9000/sonarqube` and login
10.Check that your project has been returned like in _step 3_
Actual result:projects_management page shows Carina-demo project, projects page doesn't show Carina-demo project
Expected result: projects_management page shows Carina-demo project, projects page shows Carina-demo project

<img width="1385" alt="Screen Shot 2020-08-18 at 7 16 54 PM" src="https://user-images.githubusercontent.com/32964663/90538735-9da6a480-e187-11ea-8491-36759e296f24.png">

**Comment by ilya-varchenya** [comment]
Can't start sonarqube container after step 8
Have some issue with restore of container

**Comment by ilya-varchenya** [comment]
Container's logs have a lot of similar Exceptions:
![image](https://user-images.githubusercontent.com/44428408/91699766-75eb0f80-eb7d-11ea-907b-77ad468b75c5.png)

**Comment by ilya-varchenya** [comment]
https://github.com/zebrunner/sonarqube/pull/21

**Comment by vdelendik** [comment]
closing as won't fix... it might be easier to start registrations from scratch
